mixins/localize-mixin.js

```diff
@@ -20,13 +20,21 @@ export const LocalizeMixin = superclass => class extends superclass {
 			this.__languageChangeCallback = () => {
 				if (!this._hasResources()) return;
 				const possibleLanguages = this._generatePossibleLanguages();
-				this.constructor.getLocalizeResources(possibleLanguages)
-					.then((res) => {
-						if (!res) {
+				const localizeResources = this.constructor._getAllLocalizeResources(possibleLanguages);
+				const resourcesLoadedPromise = Promise.all(localizeResources);
+				resourcesLoadedPromise
+					.then((results) => {
+						if (results.length === 0) {
 							return;
 						}
-						this.__language = res.language;
-						this.__resources = res.resources;
+						const resources = {};
+						for (const res of results) {
+							const language = res.language;
+							for (const [key, value] of Object.entries(res.resources)) {
+								resources[key] = { language, value };
```

Comment:
Interesting! So now in theory each group of resources could be in a different language.

Reply:
Yeah, it seemed like the best way to guarantee backwards compatibility while still allowing language fallbacks. I don't expect it to come up in production a lot but will likely happen a lot in dev.